targets/alert-cli/src/diff/dila/Code/__tests__/dilaDataDiff.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { describe, expect } from "@jest/globals";
+import type { Commit } from "nodegit";
+
+import ProcessCodeChanges from "../ProcessCodeChanges";
+import type { CodeFileChange } from "../types";
+import irrevelantChanges from "./dataset/filechanges_kali_code_irrevelant_changes.json";
+import noChanges from "./dataset/filechanges_kali_code_no_changes.json";
+import revelantChanges from "./dataset/filechanges_kali_code_revelant_changes.json";
+
+describe("Calcul des différences sur les conventions collectives (kali-data)", () => {
```

Comment:
Je pense que le titre est à revoir :)
```suggestion
describe("Calcul des différences sur le code (legi-data)", () => {
```